.evergreen/config.yml

```diff
@@ -213,7 +213,7 @@ tasks:
     commands:
       - func: fetch source
         vars:
-          NODE_LTS_VERSION: v18.16.0
+          NODE_LTS_VERSION: 18
```

Comment:
Ah ok, so I'll pin it for that task at npm 9.

Reply:
@durran could you also add a comment above that line while we're here explaining that the perf version must remain pinned to patch so that the time series history isn't affected by differences in the nodejs runtime performance between versions?

Reply:
Added.